File: FitnessCentar/FitnessCentar.Services/Database/Raspored.cs
using System;
using System.Collections.Generic;

namespace FitnessCentar.Services.Database;

public partial class Raspored
{
    public int Id { get; set; }

    public DateTime? DatumPocetka { get; set; }

    public DateTime? DatumZavrsetka { get; set; }

    public int? TreningId { get; set; }

    public int? TrenerId { get; set; }

    public int? AktivnostId { get; set; }

    public virtual Aktivnosti? Aktivnost { get; set; }

    public virtual ICollection<Rezervacije> Rezervacijes { get; set; } = new List<Rezervacije>();

    public virtual Treneri? Trener { get; set; }

    public virtual Treningi? Trening { get; set; }

    public DayOfWeek Dan { get; set; }
}

File: UI/fitness_mobile/lib/models/raspored.dart

import 'package:json_annotation/json_annotation.dart';

part 'raspored.g.dart';

@JsonSerializable()

class Raspored{
int? id;
DateTime? datumPocetka;
DateTime? datumZavrsetka;
int? trenerId;
int? treningId;
int? aktivnostId;
int? dan;

 Raspored(this.id,this.datumPocetka,this.datumZavrsetka,this.aktivnostId,this.trenerId,this.treningId,this.dan);




  factory Raspored.fromJson(Map<String, dynamic> json) =>
      _$RasporedFromJson(json);

  Map<String, dynamic> toJson() => _$RasporedToJson(this);

}